### ISSUE TITLE
[Feature]: No categorie by default.

### ISSUE THREAD

**Issue by YetAnotherDevWannabe** [Cidan/BetterBags#629]
### Describe the feature

I really think that by default the addon should create almost no categories.
That is the point of the plugins or manual categories.

All my gear keep getting split in 16 categories and it's sadly making the addon almost unusable since now it's taking half of my screen

Hearthstone, Gear (all) and Misc are all the categorie I think the default addon should make.

Also if I delete a categorie, it should not be instantly recreated. I deleted it for a reason...

### Describe the implementation

see above

**Comment by AMaribo** [comment]
![image](https://github.com/user-attachments/assets/711df009-a2e4-4f2e-9e85-7682ee9c7510)

**Comment by Cidan** [comment]
This is not the stated goal of this addon, and the defaults are there for a reason. Please read the instruction manual in the README for more information on how to customize.

**Comment by YetAnotherDevWannabe** [comment]
@AMaribo  Yeah I tried that and it doesn't work for me. Selected or not the category shows up and if I delete it it's back instantly.

@Cidan I did read the instructions and the problem is I can't (now idea why tho) left click the category to deactivate it (Back category for ex.). Hiding is not what I try to do since I want the item to show, just not here. Delete does work but the item stays in the category since it's recreated instantly.

Also I get it that there needs to be a minimum of default categories but damn, I think I have like 50+ of them.... And with so many of them it make finding anything worse since almost every item is in a split category of his own (especially for gear)

I personally don't care to have the gear split by slot, I find it utterly useless, but I can't seem to find anything to stop it from happening outside of manually adding each new peace of gear to a custom category, witch kinda defeats the point of having the addon altogether
